### PR TITLE
Resource status changes

### PR DESCRIPTION
**Pull request by urshala**
* Add a boolean field  `temporarily_closed` to resource model. This will inform if the resource is out of use temporarily.
* Add a boolean field `can_use_without_reservation` to resource model. This field informs whether the resource can be 
used without having any reservation.
* Add a text field `external_reservation_text` that will have further info about how the resource can be reserved in external services.
* 
external_reservation_and_free_of_use

temporarily_closed

### CODE REVIEW

File: locale/fi/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1054,6 +1057,9 @@ msgid ""
 msgstr ""
 "Ulkoisen varausjärjestelmän osoite jos tätä resurssia hallitaan muualla"
 
+msgid "External reservation free text"
+msgstr "Ulkoisen varaus ilmainen teksti"
```
Comment (tuomas777) [inline locale/fi/LC_MESSAGES/django.po]:
```suggestion
msgstr "Ulkoisen varauksen vapaatekstikenttä"
```

File: resources/models/resource.py [hunk 2]
```diff
@@ -264,6 +266,7 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
         verbose_name=_('External reservation URL'),
         help_text=_('A link to an external reservation system if this resource is managed elsewhere'),
         null=True, blank=True)
+    external_reservation_text = models.TextField(verbose_name=_('External reservation free text'), blank=True, null=True)
```
Comment (tuomas777) [inline resources/models/resource.py]: It is not [recommended](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.Field.null) to have a nullable `TextField` unless there is a specific need for that.

File: respa_admin/templates/respa_admin/resources/form/_booking.html [hunk 3]
```diff
@@ -5,6 +5,15 @@
         <h2>{% trans "Booking information" %}</h2>
         <span class="pull-right">{% trans "*Mandatory fields" %}</span>
     </div>
+    <div class="official-confirmation">
+        <span class="control-label input-label">{{ form.can_use_without_reservation.label }}</span>
+        {{ form.can_use_without_reservation }}
+        {%  if  form.can_use_without_reservation.errors %}
```
Comment (tuomas777) [inline respa_admin/templates/respa_admin/resources/form/_booking.html]:
```suggestion
        {%  if form.can_use_without_reservation.errors %}
```

File: locale/fi/LC_MESSAGES/django.po (hunk 1 above)
Comment (urshala) [reply locale/fi/LC_MESSAGES/django.po]: Changed it.

File: resources/models/resource.py (hunk 2 above)
Comment (urshala) [reply resources/models/resource.py]: Changed it.

File: respa_admin/templates/respa_admin/resources/form/_booking.html (hunk 3 above)
Comment (urshala) [reply respa_admin/templates/respa_admin/resources/form/_booking.html]: Removed the white space.

**Review by tuomas777 — APPROVED** [review]
LGTM 👍 🐷

File: locale/sv/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -1006,6 +1006,9 @@ msgstr "Högsta antalet aktiva bokningar per användare"
 msgid "Reservable"
 msgstr "Bokningsbar"
 
+msgid "Temporarily closed"
+msgstr ""
```
Comment (wiltso) [inline locale/sv/LC_MESSAGES/django.po:1010]: Tillfälligt stängt

File: respa_admin/templates/respa_admin/resources/form/_general_info.html [hunk 5]
```diff
@@ -34,4 +34,10 @@ <h4>{% trans "Contact person" %}</h4>
     <div class="external-calendar">
         <h4>{% trans "External calendar" %}</h4>
         {% include "respa_admin/forms/_input.html" with field=form.external_reservation_url %}
-</div>
+    </div>
+    <div class="external_reservation_text">
+        <h4>{% trans "External reservation free text" %}</h4>
```
Comment (wiltso) [inline respa_admin/templates/respa_admin/resources/form/_general_info.html:39]: This text doesn't really make sense it's two statements without any context, I would at least have a comma there "External reservation, free text" or even better would be something like "note from external reservation". Can you either change it or come up with something else that fits in this context.

**Comment by urshala** [comment]
This implementation is simplified and just changed the needed frontend status.

**Pull request closed without merging**